doc/translation_process.md

```diff
@@ -4,7 +4,7 @@ Translations
 The Bitcoin-Core project has been designed to support multiple localisations. This makes adding new phrases, and completely new languages easily achievable. For managing all application translations, Bitcoin-Core makes use of the Transifex online translation management tool.
 
 ### Helping to translate (using Transifex)
-Transifex is setup to monitor the Github repo for updates, and when code containing new translations is found, Transifex will process any changes. It may take several hours after a pull-request has been merged, to appear in the Transifex web interface.
+Transifex is setup to monitor the GitHub repo for updates, and when code containing new translations is found, Transifex will process any changes. It may take several hours after a pull-request has been merged, to appear in the Transifex web interface.
```

Comment:
There is a second instance of Github in this file, please fix it too (https://github.com/bitcoin/bitcoin/pull/9675/files#diff-eb26390875720fc65e1c6c74a87626ffR35).